dbt_subprojects/dex/macros/models/enrich_balancer_v3_dex_trades.sql

```diff
@@ -17,6 +17,24 @@ WITH base_trades as (
         {{ incremental_predicate('block_time') }}
     {% endif %}
 )
+, tokens_metadata as (
+    --erc20 tokens
+    select
+        blockchain
+        , contract_address
+        , symbol
+        , decimals
+    from
+        {{ tokens_erc20_model }}
+    union all
+    --native tokens
+    select
+        blockchain
+        , {{var('ETH_ERC20_ADDRESS')}} as contract_address -- 0x00..00
```

Comment:
Is this a global variable?

Reply:
yes. looks like we had in project yml file for daily/hourly/tokens/nft, but not dex 🤔  adding now